docs/migration_guide.md

```diff
@@ -0,0 +1,51 @@
+# Migrating from scikit-build
+
+```{warning}
+scikit-build-core is under active development.
+This guidance will be updated on a best-effort basis, but if you are working at the bleeding edge some of it may be out of date.
+```
+
+## Config changes
+
+- The `build-system.build-backend` key in pyproject.toml must be changed to
+  `scikit_build_core.build`.
+- Replace `scikit-build` with `scikit-build-core` in `build-system.requires`.
+- You may remove `cmake` and `ninja` from `build-system.requires` (IS THIS
+  REQUIRED? CAN THESE BE LEFT IN PLACE IF DESIRED? WHAT IF STRICTER VERSIONS
+  REQUIREMENTS ARE NEEDED THAN WHAT SCIKIT-BUILD-CORE'S CONFIG SUPPORTS?)
+- You must fill out the `tool.scikit-build` table in pyproject.toml, see
+  [getting started](./getting_started.md) for more information.
```

Comment:
A great advanced trick to go from setup.py/setup.cfg to pyproject.toml quickly is to change `skbuild` into `setuptools`, use `hatch new --init` to convert, then change `hatchling` back into `scikit-build-core`.

Reply:
This is mainly useful if you have everything in setup.py instead of pyproject.toml, right?

Reply:
Well, you can't put everything in pyproject.toml if you are using scikit-build classic. It requires some values be present in `setup.py`. But yes, it's mostly useful to move things from setup.cfg/setup.py to pyproject.toml.